CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed 
```

Comment:
There's already a "Changed" section below. Please add this entry to that section instead.